### PR TITLE
docs: document how the deployments are triggered

### PR DESCRIPTION
**Pull request by sandhose**
[Rendered](https://github.com/zazuko/cube-creator/blob/sandhose/deployment-readme/README.md#deployment)

### CODE REVIEW

**Comment by martinmaillard** [comment]
The explanation looks good but I wonder if it should be in this repo. Everything about the github actions fits, but infos about deployment should probably not be tied to this repo. Maybe put them in the BAFU wiki?

**Comment by sandhose** [comment]
Sure. @ludovicm67 and I will touch the GitHub Actions soon to align with what we do in other projects, so I'll just add what gets deployed in the wiki and hold this for now

**Comment by l00mi** [comment]
@tpluscode can you please add one line how we easiest create the semver git tags? Do you do it manually or what tool?

**Comment by tpluscode** [comment]
@l00mi right now it's manual but I'm talking to @sandhose to improve the CI process to automate

**Comment by l00mi** [comment]
@tpluscode I think it is not necessary. It is okay manual.. Just put in the document which 2-3 commands you type.

**Comment by tpluscode** [comment]
manual is not ok. trust me, I'm an engineer 😆

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ Commit messages usually follow the guidelines from [Conventional Commits](https:
...
+## Release process
+
+All notable changes to the project(s) should be documented using [@atlassian/changesets](https://github.com/atlassian/changesets). When preparing a PR, run `yarn changeset` in the repository which will ask what changed, the affected packages, and create a small markdown file which must be committed to the repository.
+
+Alternatively, that file can be created directly in the browser, courtesy of [Changesets bot](https://github.com/changesets/bot) which creates PR comment summarising the changes included in the PR in question.
+
+Releases are managed automatically by [GitHub actions](.github/workflows/releases.yaml). As PRs get merged to master, the first job creates or updates a pull request which bumps package versions according to the combined information from all accumulated changeset files. Once merged, the second job kicks in which tags the repository and triggers INT deployment.
+
+The package versions can also be bumped manually by running `yarn changeset version` in the repository and committing the result. When pushed, the new versions will be picked up by the aforementioned GitHub workflow job to tag the respository.
+
```
Comment (tpluscode) [inline README.md:53]: @martinmaillard

**Comment by l00mi** [comment]
@tpluscode I don't understand what is missing, waiting here?

**Comment by tpluscode** [comment]
@l00mi I don't know anymore. The only difference from `master` is in fact the **Deployment** section which we don't want?

If so, the we can close without merging

**Comment by l00mi** [comment]
Okay, but then can you please describe how the deployment is set-up currently. Which actions and things make it happen. Or asked otherwise what would be the things needed to do for a second deployment?

**Pull request closed without merging**